client/components/account-info.jsx

```diff
@@ -37,6 +38,7 @@ export default class AccountInfo extends React.Component {
           window.location.hash = 'login';
         } else if (action === 'sign-in') {
           window.localStorage.setItem('token', result.token);
+          window.localStorage.setItem('username', user);
```

Comment:
Instead of storing the username in localStorage, you should be able to pass it from your App component's state to other components. We want to avoid "sharing" things between components using anything other than props or context.

Reply:
You can remove line 43 now

```js
          window.localStorage.setItem('username', user);

```